src/include/os_fs.i

```diff
@@ -114,7 +114,8 @@ __wt_fs_remove(WT_SESSION_IMPL *session, const char *name, bool durable)
 
 	file_system = S2C(session)->file_system;
 	wt_session = (WT_SESSION *)session;
-	ret = file_system->fs_remove(file_system, wt_session, path, durable);
+	ret = file_system->fs_remove(
+	    file_system, wt_session, path, durable ? WT_FS_DURABLE : 0);
```

Comment:
A primary reason for this change is to ensure all of us think about durability when we call the remove/rename functions, and I think a flag makes that less likely.